lib/double_entry/validation/line_check.rb

```diff
@@ -89,9 +89,18 @@ def line_error_message(line, previous_line, previous_balance)
           END_OF_MESSAGE
       end
 
-      def cached_balance_correct?(account)
+      def cached_balance_correct?(account, log)
         DoubleEntry.lock_accounts(account) do
-          return AccountBalance.find_by_account(account).balance == account.balance
+          account_balance_1 = AccountBalance.find_by_account(account).balance
+          account_balance_2 = account.balance
+          correct = (account_balance_1 == account_balance_2)
+          log << <<~MESSAGE unless correct
+            *********************************
+            Error on account #{account} :: balance: #{account_balance_1} != #{account_balance_2}
```

Comment:
I'm wondering if this message is clear enough. Would it be worth stating which one is the cached value and which is the running balance?

Reply:
yeah, that a good idea